site/openapiSchema.yaml

```diff
@@ -0,0 +1,521 @@
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: DropOffName
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: DropOffAddress
+          in: query
+          required: false
+          schema:
+            type: array
+            items:
+              type: string
+        - name: DropOffTown
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: DropOffCounty
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: DropOffPostcode
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: DropOffCountryCode
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: DropOffLongitude
+          in: query
+          required: false
+          schema:
+            type: number
+        - name: DropOffLatitude
+          in: query
+          required: false
+          schema:
+            type: number
+        - name: FromDate
+          in: query
+          required: true
+          schema:
+            type: string
+            format: date
+        - name: FromTime
+          in: query
+          required: true
+          schema:
+            type: string
+            pattern: '^\d{4}$'
+        - name: ReturnDate
+          in: query
+          required: false
+          schema:
+            type: string
+            format: date
+        - name: ReturnTime
+          in: query
+          required: false
+          schema:
+            type: string
+            pattern: '^\d{4}$'
+        - name: Adults
+          in: query
+          required: true
+          schema:
+            type: integer
+        - name: Children
+          in: query
+          required: false
+          schema:
+            type: integer
+        - name: Infants
+          in: query
+          required: false
+          schema:
+            type: integer
+        - name: OutFlight
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: ReturnFlight
+          in: query
+          required: false
+          schema:
+            type: string
+      responses:
+        '200':
+          description: A list of available transfer products
+        '400':
+          description: Invalid request parameters
+        '401':
+          description: Missing token or authentication details
+        '403':
+          description: Invalid agent credentials
+        '404':
+          description: Data not found (e.g. booking, location)
+        '405':
+          description: Method not allowed
+        '422':
+          description: Validation error (e.g. missing or invalid parameters)
+        '500':
+          description: Internal server error or upstream service unavailable
+        '9999':
```

Comment:
This section appears to mix up HTTP response codes with the error codes returned in the response.

I think that's not helpful for anyone attempting to use this schema. It needs to be clear that for most errors (other than HTTP/network level ones), we'll actually be returning 200 OK and returning the error code in the response.